VideoWeb/VideoWeb/ClientApp/src/app/shared/mappers/participant-panel-model-mapper.ts

```diff
@@ -59,6 +60,9 @@ export class ParticipantPanelModelMapper {
         });
         const allMapped = linked.map(l => new ParticipantPanelModel(l));
         allMapped.push(new ParticipantPanelModel(participant));
-        return allMapped;
+        return [
```

Comment:
Can you add tests for this?

Reply:
As far as I am aware we cannot test private methods so this mapper would have to go public/protected

Reply:
I'll give you a call.

Reply:
unit test added